conf/defaults.ini

```diff
@@ -867,8 +867,8 @@ max_attempts = 3
 min_interval = 10s
 
 [unified_alerting.screenshots]
-# Enable screenshots in notifications. This option requires a remote HTTP image rendering service. Please
-# see [rendering] for further configuration options.
+# Enable screenshots in notifications. This option requires either the Grafana Image Renderer plugin.
+# Please see [rendering] for further configuration options.
 capture = false
```

Comment:
```suggestion
# Enable screenshots in notifications. This option requires either the Grafana Image Renderer plugin.
# For more information on configuration options, refer to [rendering].
capture = false
```

Reply:
Hey George-this is our standard phrasing for when you want to give more info...and we avoid using please:)